accepted/future-releases/spread-collections/implementation-plan.md

```diff
@@ -0,0 +1,145 @@
+# Implementation Plan for "Spread Collections"
+
+Owner: rnystrom@google.com ([@munificent](https://github.com/munificent/) on GitHub)
+
+Relevant links:
+
+* [Tracking issue](https://github.com/dart-lang/language/issues/47)
+* [Proposal](https://github.com/dart-lang/language/blob/master/accepted/future-releases/spread-collections/feature-specification.md)
+
+## Phase 0 (Prerequisite)
+
+### Experiment support
```

Comment:
Let's just take this as given, delete this section.

Reply:
Done.